mcpwm_foc.c

```diff
@@ -2773,7 +2775,7 @@ void mcpwm_foc_adc_int_handler(void *p, uint32_t flags) {
 
 		// Set motor phase
 		{
-			if (!motor_now->m_phase_override) {
+			if (!motor_now->m_phase_override & !(motor_now->m_control_mode == CONTROL_MODE_OPENLOOP_PHASE)) {
```

Comment:
As long as you're modifying this section, could you drop a comment about what is being tested and why?